src/seedsigner/views/screensaver.py

```diff
@@ -96,8 +96,6 @@ def _render(self):
         # Display version num below SeedSigner logo
         font = Fonts.get_font(GUIConstants.get_body_font_name(), GUIConstants.get_top_nav_title_font_size())
         version = f"v{controller.VERSION}"
-
-        # The logo png is 240x240, but the actual logo is 70px tall, vertically centered
```

Comment:
Don't remove explanatory comments. Why is `logo_height` set to 70? Should not be left as a mystery.